client/src/api/auth.js

```diff
@@ -14,9 +30,17 @@ export const register = ({ email, name, division }) => {
     register: true
   })
     .then(resp => {
-      if (resp.kind === 'verifyEmail') {
-        localStorage.setItem('token', resp.data.authToken)
-        localStorage.setItem('teamToken', resp.data.teamToken)
+      switch (resp.kind) {
+        case 'verifyEmail':
+          localStorage.setItem('token', resp.data.authToken)
+          localStorage.setItem('teamToken', resp.data.teamToken)
+
+          return route('/profile')
```

Comment:
Is this correct? I don't see the verifyEmail response kind; should be "goodVerify"? Also, this is only when the server is not in verifyEmail mode, if response is goodVerifySent, route them to a check your email page?